test/api.mocha.coffee

```diff
@@ -350,6 +356,11 @@ describe "API", ->
 
               # Send them invitations
               (_party, cb) ->
+                 # @TODO - _party should be array of newly
```

Comment:
Undesirable. That darned database is already huge. We have more than 30 users now. 

Thanks for mentioning that! It might explain some of the oddness I see in there, which I haven't finished investigating yet. You might have saved me some work. :)